src/components/RoleSelectOption/RoleSelectOption.tsx

```diff
@@ -36,70 +37,43 @@ const optionWrapperStyles = css`
   border-radius: 6px;
 `
 
-export const Option: React.FC<OptionProps> = ({ label, text, src, alt }) => {
+export const Option: React.FC<OptionProps> = ({
+  label,
+  text,
+  src,
+  hoveredSrc,
+  alt,
+}) => {
+  const [isHovered, setIsHovered] = useState(false)
+
   return (
     <RoleSelectWrapper>
-      <div
-        className={
-          label === 'Buyer'
-            ? 'inner-select-wrapper-buyer'
-            : 'inner-select-wrapper-seller'
-        }>
-        <div
-          className={
-            label === 'Buyer'
-              ? 'dark-polygon-wrapper-bg'
-              : 'white-polygon-wrapper-bg'
-          }>
-          <img src={src} alt={alt} />
+      <RoleSelectBackground
+        onMouseEnter={() => setIsHovered(true)}
+        onMouseLeave={() => setIsHovered(false)}>
+        <div className='polygon-bg-image'>
```

Comment:
We are using styled components so no class names assignments.
Please extract div into separate styled one